Stresser/ProcessArtifact.cpp

```diff
@@ -38,13 +39,19 @@ void ProcessArtifact::install() const
         throw Win32ErrorCodeException("Failed to create a process!");
     }
 
-    AutoHandle autoProcessHandle(processInfo.hProcess);
+    AutoHandle autoHProcess(processInfo.hProcess); 
     AutoHandle autoThreadHandle(processInfo.hThread);
+    this->pid = processInfo.dwProcessId;
 }
 
-void ProcessArtifact::uninstall() const
+void ProcessArtifact::uninstall()
 {
-    Sleep(2000);
+    const AutoHandle autoProcess(OpenProcess(SYNCHRONIZE, FALSE, this->pid));
```

Comment:
Please document `SYNCHRONIZE` - I think you used it to solve the problem with data races (???)